lib/govuk_app_config/rails_ext/action_dispatch/debug_exceptions.rb

```diff
@@ -0,0 +1,53 @@
+require "action_dispatch/middleware/debug_exceptions"
+
+module GovukLogging
+  module RailsExt
+    module ActionDispatch
+      def self.should_monkey_patch_log_error
```

Comment:
Super minor: I think the convention for methods that return a Boolean is that they end in a question mark.